src/UI/Actions.hs

```diff
@@ -844,9 +845,11 @@ updateBrowseFileContents contents s =
 applySearch :: (MonadIO m) => AppState -> m AppState
 applySearch s = do
   r <- runExceptT (Notmuch.getThreads searchterms (view (asConfig . confNotmuch) s))
+  t <- liftIO (zonedTimeToUTC <$> getZonedTime)
   case r of
     Left e -> pure $ setError e s
-    Right threads -> updateList threads <$> notifyNumThreads s threads
+    Right threads -> set asLocalTime t
+                     <$> (updateList threads <$> notifyNumThreads s threads)
```

Comment:
surely `set asLocalTime t . updateList threads <$> notifyNumThreads s threads` ... ?

Reply:
Sometimes my representation of `fmap` as being something I can loop over containers in an abstract way other than a general abstraction keeps me from doing the right thing here. I read it as set the local time for each thread instead of a better `>>=`... ah well